.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,5 @@
-*Issue #, if available:*
+<!-- Provide summary of changes -->
 
-*Description of changes:*
+<!-- Issue number, if available. E.g. "Fixes #31", "Addresses #42, 77" -->
 
-
-By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
+By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
```

Comment:
are we allowed to edit this text?

Reply:
Oh, I thought that was the most updated text, since I see it on the [v1 CLI](https://github.com/aws/amazon-ecs-cli/blob/mainline/.github/PULL_REQUEST_TEMPLATE.md) and [CDK](https://github.com/aws/aws-cdk/blob/master/.github/PULL_REQUEST_TEMPLATE.md).